frontend/src/components/object/ObjectUploadBasic.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+
+import { Button, useConfirm, useToast } from '@/lib/primevue';
+import { useObjectStore } from '@/store';
+
+import type { Ref } from 'vue';
+
+// Props
+type Props = {
+  bucketId: string,
+  objectId: string
+};
+
+const props = withDefaults(defineProps<Props>(), {});
+
+// Emits
+const emit = defineEmits(['on-file-changed', 'on-file-uploaded']);
+
+// Store
+const objectStore = useObjectStore();
+
+// State
+const fileInput: Ref<any> = ref(null);
+const file: Ref<File | undefined> = ref(undefined);
+
+// Actions
+const confirm = useConfirm();
+const toast = useToast();
+
+const confirmUpdate = () => {
+  confirm.require({
+    message: 'Please confirm that you want to upload a new version.',
+    header: 'Upload new version',
+    acceptLabel: 'Confirm',
+    rejectLabel: 'Cancel',
+    accept: () => onUpload(),
+    reject: () => {
+      // Intentionally left empty
+    },
```

Comment:
Does this attribute need to be defined with an empty callback? Would be ideal to just not have it even defined at all if possible.